README.md

```diff
@@ -166,6 +166,13 @@ Semantic HTML is the use of HTML markup to reinforce the semantics, or meaning,
 * Have you ever used JavaScript templating?
   * If so, what libraries have you used?
 * Explain "hoisting".
+    Hoisting is the lifting of variables and functions up to the very top (to global reach or function coverage).
```

Comment:
nie do końca, co z tego, że użyjesz hoistowaną zmienną, jeśli defaultowo jest undefined?
https://scotch.io/tutorials/understanding-hoisting-in-javascript